### PR TITLE
Fixes #30558 - Fix typos in templates DSL

### PR DESCRIPTION
**Pull request by mirekdlugosz**
Fix typos found while reviewing templates DSL docs.

CC @ares

### CODE REVIEW

File: lib/foreman/renderer/scope/macros/base.rb [hunk 1]
```diff
@@ -140,7 +140,7 @@ def indent(count, skip1: false)
           apipie :method, 'Performs a DNS lookup on Foreman server' do
             required :name_or_ip, String, desc: 'a hostname or IP address to perform DNS lookup for'
             returns String, desc: 'IP resolved via DNS if hostname was given, hostname if an IP was given.'
-            raises error: Timeout::Error, desc: 'when DNS resolve could not be performed in time set by global setting `dns_confict__timeout`'
+            raises error: Timeout::Error, desc: 'when DNS resolve could not be performed in time set by global setting `dns_timeout`'
```
Comment (mirekdlugosz) [inline lib/foreman/renderer/scope/macros/base.rb:143]: This one I'm not 100% sure about. `confict__` looks like something generated by script, but is only instance in foreman source.
`confict` does sound similar to `conflict`, but there are no references to DNS conflict whatsoever.

Meanwhile, `dns_timeout` is referenced just couple of lines below, so I'm assuming this is what was meant here.

File: app/assets/javascripts/locale/cs_CZ/app.js [hunk 2]
```diff
@@ -1 +1 @@
...
...stroje"],"Something went wrong while selecting hosts - %s":["Při vybírání strojů se něco pokazilo – %s"],"%s selected hosts":["vybráno %s strojů"],"The following hosts were not %{action}: %{missed_hosts}":[""],"No or invalid power state selected!":[""],"The %{proxy_type} proxy of the selected hosts was cleared":[""],"No templates found":["Nenalezeny žádné šablony"],"HTTP Proxy connection successful.":["Připojení k HTTP proxy úspěšné."],"%{user} Downloaded %{key} as pem file":["%{user} Stažen %{key} v podobě pem souboru"],"Successfully recreated":[""],"Could not recreate a new SSH key":["Nedaří se znovu vytvořit nový SSH klíč"],"Successfully delete %s":["Úspěšně smazat %s"],"Could not delete %s":["%s se nedaří smazat"],"Successfully overridden all parameters of Puppet class %s":[""],"Successfully reset all parameters of Puppet class %s to their default values":[""],"No parameters to override for Puppet class %s":[""],"Report is going to be rendered at %s and it will be delivered via e-mail once ready.":[""],"Report is being rendered, it will be delivered via e-mail.":["Výkaz je vykreslován, bude doručen prostřednictvím e-mailu."],"Could not generate the report, check the form for er
```
Comment (ares) [inline app/assets/javascripts/locale/cs_CZ/app.js]: this one is I believe generated, please drop it from this PR

File: lib/foreman/renderer/scope/macros/base.rb (hunk 1 above)
Comment (ares) [reply lib/foreman/renderer/scope/macros/base.rb:143]: correct, the setting is `dns_timeout`

**Review by ares — CHANGES_REQUESTED** [review]
Thanks @mirekdlugosz, just one nit, but happy to merge afterwards! Good stuff!

File: app/assets/javascripts/locale/cs_CZ/app.js (hunk 2 above)
Comment (mirekdlugosz) [reply app/assets/javascripts/locale/cs_CZ/app.js]: Dropped from commit and force-pushed.

File: locale/cs_CZ/foreman.po [hunk 3]
```diff
@@ -10450,7 +10450,7 @@ msgid "You are using an unsupported browser."
 msgstr "Používáte nepodporovaný prohlížeč."
 
 msgid "You can add SSH public keys to a user in Foreman."
-msgstr "Ve Formean je možné přidat k uživateli veřejné části SSH klíčů."
+msgstr "Ve Foreman je možné přidat k uživateli veřejné části SSH klíčů."
```
Comment (adamruzicka) [inline locale/cs_CZ/foreman.po]: Not sure if we should be changing the translations directly, wouldn't it get overwritten on the next translation sync?

Comment (ares) [reply locale/cs_CZ/foreman.po]: yum, that's the same comment as for the js file, please drop this one from the PR and we're good to go :-)

Comment (mirekdlugosz) [reply locale/cs_CZ/foreman.po]: Done!

**Review by adamruzicka — APPROVED** [review]
LGTM

**Review by ares — APPROVED** [review]
Thanks @mirekdlugosz and @adamruzicka, merging

**Pull request merged**